runelite-client/src/test/java/net/runelite/client/plugins/entityhider/EntityHiderPluginTest.java

```diff
@@ -72,7 +72,6 @@ public void before()
 	{
 		Guice.createInjector(BoundFieldModule.of(this)).injectMembers(this);
 
-		when(client.getIgnoreContainer()).thenReturn(ignoreNameableContainer);
```

Comment:
I'd advise changing this to a lenient stub so future tests can't run into a case where they'd need to stub this just to avoid NPEs.

Reply:
Resolved in 79579366117a37bd2c2245ed34ad81c84a3f3db3